guide/12-enrich-data-with-thematic-information/part4_discover_and_enrich_standard_geographies.ipynb

```diff
@@ -0,0 +1,1208 @@
+{
```

Comment:
<p>I feel this intro can be rephrased to be more concise. I can't follow the thread of thought easily as to what this guide will do, I suggest making it the first sentence and then explaining.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>